app/serializers/order_serializer.rb

```diff
@@ -61,10 +64,34 @@ def ordered_by_current_user
   def from_today
     object.from_today?
   end
+  
+  def who_ordered
+    object.user.name
+  end
+
+  def current_user_dish
+    if current_user_ordered_dish?
+      object.dishes.where('dishes.user_id = ?', current_user.id).first.name
+    else
+      "You didn't order any dish."
+    end
+  end
+  
+  def current_user_dish_price
+    if current_user_ordered_dish?
+      object.dishes.where('dishes.user_id = ?', current_user.id).first.price.to_s
```

Comment:
Since we're making the same query twice (here and in current_user_dish method), we could cache the current user dish like so: 

```
def current_user_dish
  @current_user_dish ||= object.dishes.find_by(user: current_user)
end
```

And then use it like `current_user_dish.price.to_s` / `current_user_dish.name`.